transcom/mymove
# INT-B-20001

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A949639&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

PPTAS Reports requires Postman to test

### How to test

1. Import PPTAS api into Postman
2. Create a Navy move with an HHG and complete the entire move/payment process
3. http://primelocal:3000/prime/pptas/moves?since=2021-07-23T18:30:47.116Z
4. Check the report has the required details from the sheet linked below


[PPTAS working spreadsheet (27 June 2024) Jays Comments Dan Notes.xlsx](https://github.com/user-attachments/files/16183536/PPTAS.working.spreadsheet.27.June.2024.Jays.Comments.Dan.Notes.xlsx)
